index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="./reset.css" />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="/">
+          <img src="./img/Property 1=lg.png" alt="logo" />
+        </a>
+        <button><a href="./login.html">로그인</a></button>
```

Comment:
❗️ 수정요청
로그인 버튼을 눌렀을 때 './login.html'로 이동시켜주는 역할을 하니, 앵커 태그만 남겨두셔도 될 것 같습니다~
버튼 요소처럼 보여도 꼭 버튼 태그로 작성하실 필요는 없습니다! 
해당 태그의 역할에 따라 태그를 작성하시는 것이 더 시맨틱하게 작성하는 방법입니다.

```suggestion
        <a href="./login.html">로그인</a>
```